.eslintignore

```diff
@@ -1,4 +1,5 @@
 dist/
 [Tt]emp/
 [Ss]ample/
+samples/
```

Comment:
Think the previous line should be fixed, it was a typo on my part that I was fixing when you landed the PR and then I forgot about it